.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master,  releasebranch_7_8]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    # Check every Saturday at 18:36
+    - cron: '36 18 * * 6'
+
+jobs:
+  analyze:
+    name: ${{ matrix.language }} - Analyze
```

Comment:
I don't see reason for " - Analyze" here. Isn't that just a clutter since there is nothing else than analyze? It doesn't seem to add much info to CodeQL if you don't already know what that is. _CodeQL / cpp_ in checks seems quite sufficient to me, but feel free to modify. For flake.yml, I just rewrote it as _Python Flake8 Code Quality_ / {directory}.

Reply:
Same here - `name: Analyze` appears as default name during auto-generation. For me either way is fine. Do you mean `python / cpp` above?